### DISCUSSION TITLE
Previously deleted "Music when" will not be removed.

### DISCUSSION THREAD

**Discussion by lesjtay** [pnp/powershell#446] [Q&A]
So if I can't make this run it must be a bug ;-) Arrogance aside any help would be much appreciated.

PS> Get-PnPDeletedMicrosoft365Group

DisplayName Id ----------                                 HasTeam SiteUrl

muscic26 ----------------- af1baf8d-4604-46c8-be46-bb74c73be30c False
muscic27 ----------------- ded85681-182b-4d58-a2da-cf73c121872d False
Music  when -------------- 1fed85b2-f275-4a68-b439-584222415857 False

PS> $group = Get-PnPDeletedMicrosoft365Group -Identity 1fed85b2-f275-4a68-b439-584222415857
PS>> Remove-PnPDeletedMicrosoft365Group -Identity $group

PS> Get-PnPDeletedMicrosoft365Group

DisplayName Id -----------                                  HasTeam SiteUrl
                                
muscic26 ------------------ af1baf8d-4604-46c8-be46-bb74c73be30c False
muscic27 ------------------ ded85681-182b-4d58-a2da-cf73c121872d False
Music  when ----------------1fed85b2-f275-4a68-b439-584222415857 False

PnP.PowerShell v1.4.6 -nightly

**Comment by veronicageek** [comment]
@lesjtay - Definitely not a bug, I have no issue with v.1.4.6

![image](https://user-images.githubusercontent.com/25181757/111323848-245e2a00-8662-11eb-91f5-a5065a25387e.png)

**Comment by lesjtay** [comment]
Hi Veronique,

I am far from knowledgeable enough to actually think it is a bug. Attempted humor. However I would like to know what I’m doing wrong.

Please help me if you have time. I will be back to a keyboard at 4:00 pm pst.

Les

Sent from my iPhone

On Mar 16, 2021, at 7:17 AM, Veronique Lengelle ***@***.***> wrote:

﻿

@lesjtay<https://github.com/lesjtay> - Definitely not a bug, I have no issue with v.1.4.6

[image]<https://user-images.githubusercontent.com/25181757/111323848-245e2a00-8662-11eb-91f5-a5065a25387e.png>

**Reply by lesjtay** [reply]
Thank you for setting it up for a look. First off I see that my manual formatting should have left the Id header with the Ids below it.

The "Music when" site shows "Has Team" as False could that have something to do with my problem?

My $group variable shows:

DisplayName---Id-----------------------------------------------                                   HasTeam--- SiteUrl

Music  when----1fed85b2-f275-4a68-b439-584222415857----False

Can't see your variable contents but did I dirty mine?

Again any help is a kindness Veronique,

Les

**Comment by veronicageek** [comment]
Nope @lesjtay, it doesn't change anything. The group is still deleted with no issue for me (still v1.4.6). 

![image](https://user-images.githubusercontent.com/25181757/111514266-7928a000-8749-11eb-9483-c142e0914fba.png)

Maybe try to create a new group and delete it (for testing purposes)?

**Comment by lesjtay** [comment]
Hi Veronique,

It is very nice of you to spend so much time trying to duplicate my issue.

“Music when” is just one of several preexisting sites that I have deleted soft and then tried to hard delete.

Curious though about the difference between a group and a site created with a group?

Les

Sent from my iPhone

On Mar 17, 2021, at 10:56 AM, Veronique Lengelle ***@***.***> wrote:

﻿

Nope @lesjtay<https://github.com/lesjtay>, it doesn't change anything. The group is still deleted with no issue for me (still v1.4.6).

[image]<https://user-images.githubusercontent.com/25181757/111514266-7928a000-8749-11eb-9483-c142e0914fba.png>

Maybe try to create a new group and delete it (for testing purposes)?
